mk/gen_ld_sections.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+
+# Input
+CONFIG_LDS_H=$1
+# Output
+SECTIONS_LDS_H=$2
+
+rm -f $SECTIONS_LDS_H && touch $SECTIONS_LDS_H
+
+SECTIONS=`grep -o 'LDS_SECTION_VMA_[0-9a-zA-Z_]*' $CONFIG_LDS_H | \
+			sed 's/LDS_SECTION_VMA_//g' | uniq`
```

Comment:
I guess `sort -u` is more appropriate than `uniq`

Reply:
Sort would change the sections order.